### PR TITLE
add topNav pipe between 'Sheet' and 'Projects'

### LINKED ISSUE #677: Add pipe in top navigation between time,sheet and project,report

### PR DESCRIPTION
**Pull request by Slash52**
closes #677

### CODE REVIEW

File: apps/web/frontend/components/topNavigation/topNavigation.tsx [hunk 1]
```diff
@@ -33,14 +33,12 @@ export const TopNavigation = (): JSX.Element => {
         {session.status !== 'authenticated' && (
           <TopNavigationLink onClick={() => signIn('github')}>Sign in</TopNavigationLink>
         )}
-
         <TopNavigationLink href="/time">Time</TopNavigationLink>
         <TopNavigationLink href="/sheet">Sheet</TopNavigationLink>
+        <div className="h-[20px]  w-[1px] bg-blue-400" />
```
Comment (PascalHelbig) [inline apps/web/frontend/components/topNavigation/topNavigation.tsx:38]: Please use `w-px` instead of `w-[1px]`

Comment (PascalHelbig) [inline apps/web/frontend/components/topNavigation/topNavigation.tsx:38]: `h-[20px]`: arbitrary values should be avoided if possible. 

If you need the height to display anything at all, you can alternatively use CSS Flexbox to achieve that as well. The pipe is a child of a flexbox. You can tell a child to "stretch" the height.
More information here: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

If this makes the pipe too high, you can use vertical margin to adjust it again: e.g. `my-1`

Comment (PascalHelbig) [inline apps/web/frontend/components/topNavigation/topNavigation.tsx:38]: The pipe looks a little bit weird on light mode:
image

Please make it gray on light mode.

**Review by PascalHelbig — COMMENTED** [review]
Looks good, but some smaller comments

**Comment by Slash52** [comment]
closes #677

**Pull request merged**